OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -1565,8 +1595,20 @@ public void run() {
                return;
             }
 
+            if (remoteParams != null && remoteParams.useEmailAuth && externalIdAuthHash == null) {
+               String errorMessage = "External Id authentication (auth token) is set to REQUIRED for this application. Please provide an auth token from your backend server or change the setting in the OneSignal dashboard.";
+               if (completionCallback != null)
+                  completionCallback.onFailure(new ExternalIdError(ExternalIdErrorType.REQUIRES_EXTERNAL_ID_AUTH, errorMessage));
+               Log(LOG_LEVEL.ERROR, errorMessage);
+               return;
+            }
+
+            String internalExternalIdAuthHash = externalIdAuthHash;
```

Comment:
Can we change the name of internalExternalIdAuthHash? Is the only change we are making is making it all lower case? If that is the case maybe include the lowercase aspect in the name?